src/include/splash/basetypes/ColTypeDim.hpp

```diff
@@ -51,6 +56,35 @@ namespace splash
         {
             return Dimensions::getSize();
         }
+
+        std::string toString() const
+        {
+            return "Dimension";
```

Comment:
ColTypeDimArray is "DimArray" but ColTypeDim is "Dimension", you should decide on one style